src/test_workflow/benchmark_test/benchmark_args.py

```diff
@@ -0,0 +1,118 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+import argparse
+import json
+import logging
+from typing import IO
+
+from test_workflow.test_jsonargs import JsonArgs
+
+
+# Contains the arguments required to run a perf test.
+class BenchmarkArgs:
+    bundle_manifest: IO
+    stack_suffix: str
+    config: IO
+    keep: bool
+    insecure: bool
+    single_node: bool
+    min_distribution: bool
+    manager_node_count: int
+    data_node_count: int
+    client_node_count: int
+    ingest_node_count: int
+    ml_node_count: int
+    data_node_storage: int
+    ml_node_storage: int
+    jvm_sys_props: str
+    additional_config: str
+    workload: str
+    benchmark_config: IO
+    user_tag: str
+    target_hosts: str
+    logging_level: int
+
+    def __init__(self) -> None:
+        parser = argparse.ArgumentParser(description="Test an OpenSearch Bundle")
+        parser.add_argument("--bundle-manifest", type=argparse.FileType("r"), help="Bundle Manifest file.",
+                            required=True)
+        parser.add_argument("--suffix", dest="suffix", help="Suffix to be added to stack name for performance test")
+        parser.add_argument("--component", dest="component", default="OpenSearch",
+                            help="Component name that needs to be performance tested")
+        parser.add_argument("--config", type=argparse.FileType("r"), help="Config file.", required=True)
+        parser.add_argument(
+            "--without-security", dest="insecure", action="store_true",
+            help="Force the security of the cluster to be disabled.", default=False)
+        parser.add_argument("--keep", dest="keep", action="store_true",
+                            help="Do not delete the working temporary directory.")
+        parser.add_argument("--single-node", dest="single_node", action="store_true",
+                            help="Is this a single node cluster")
+        parser.add_argument("--min-distribution", dest="min_distribution", action="store_true", help="Is it the "
```

Comment:
Put each argument such as `help` on its own line and it won't need to be wrapped like this.

Reply:
Done!